### ISSUE TITLE
EdgeRouter: Unable to do IPv6 masquade on 0.3.12

### ISSUE THREAD

**Issue by samip5** [yggdrasil-network/yggdrasil-go#642]
Hello there,

Thank you for this awesome sounding project. I tried to install this onto a EdgeRouter-X running firmware v.2.0.6 but unfortunately the following fails:
```
set interfaces yggdrasil tun1 masquerade from 2a01:4f9:cxxf:xx::/64
```

The error is:
```
ip6tables v1.4.21: can't initialize ip6tables table `nat': Table does not exist (do you need to insmod?)
Perhaps ip6tables or your kernel needs to be upgraded.
```

Of course there is no NAT table as IPv6 has no NAT, so what's up with that?

**Comment by neilalexander** [comment]
The IPv6 NAT module is present on the 1.x branch, so I’m surprised to hear it’s been removed from 2.x. I’ll test this too.

**Comment by neilalexander** [comment]
It looks like this may have been fixed in v2.0.8 firmware. Can you try with that firmware and let me know?

**Comment by samip5** [comment]
Yes, that worked so upgrading to v2.0.8 fixes the problem.

I have never even heard of using NAT with IPv6.
